### PR TITLE
looker crawler stuck in LookML parsing [sc-25902]

### PR DESCRIPTION
**Pull request by alyiwang**
### 🤔 Why?

sql-metadata parser may stuck at certain SQL queries.

### 🤓 What?

- don't do SQL parsing at crawler, but send the raw query in MCE for server-side processing.

### 🧪 Tested?

Unit tests
tested against Looker instance, and verified the MCE

### ☑️ Checks

- [x] My PR contains actual code changes, and I have updated the version number in `pyproject.toml`.

### CODE REVIEW

File: metaphor/looker/lookml_parser.py [hunk 1]
```diff
@@ -159,49 +161,29 @@ def _get_upstream_datasets(
             if base_view_name is None:
                 base_view_name = explore_name
 
-            upstreams.update(
-                _get_upstream_datasets(base_view_name, raw_model, connection)
+            query = LookerViewQuery(
```
Comment (usefulalgorithm) [inline metaphor/looker/lookml_parser.py:164]: is it possible to have both `sql` and `explore_source` keys in `derived_table`? If so then `query` gets replaced, is this ok?

File: metaphor/looker/lookml_parser.py [hunk 2]
```diff
@@ -159,49 +161,29 @@ def _get_upstream_datasets(
...
+                query="_",
```
Comment (usefulalgorithm) [inline metaphor/looker/lookml_parser.py:165]: what does this mean? is this liquid syntax?

Comment (alyiwang) [reply metaphor/looker/lookml_parser.py:165]: this is meant to be fixed by https://github.com/MetaphorData/app/pull/8269

File: metaphor/looker/lookml_parser.py (hunk 1 above)
Comment (alyiwang) [reply metaphor/looker/lookml_parser.py:164]: The derived table should have either one, but shouldn't be both.

File: metaphor/looker/lookml_parser.py (hunk 2 above)
Comment (alyiwang) [reply metaphor/looker/lookml_parser.py]: updated

**Review by usefulalgorithm — APPROVED** [review]
nice!

**Pull request merged**